skame/schemas/base.py

```diff
@@ -232,7 +238,7 @@ def _validate(self, data: dict, fields: dict, value_getter: "function") -> dict:
                 value = value_getter(data, field)
                 cleaned_value = self.mapping[field].validate(value)
             except KeyError:
-                errors[str(field)] = _("Field `{0}` is required.").format(field)
+                errors[str(field)] = self.messages.get('required', type(self).messages['required']).format(field)
```

Comment:
I think that should be written in more simple manner. 

I think the best approach for that would be, define one unique messages property in the constructor that merges the default ones and the user provided (if them are not None), and here just access with `self.messages['required']`.